platform/mellanox/mlnx-platform-api/sonic_platform/psu.py

```diff
@@ -112,7 +112,9 @@ def __init__(self, psu_index, platform):
 
         else: 
             logger.log_info("Not reading PSU{} VPD data: Platform is fixed".format(self.index))
-
+            self.model = "N/A"
```

Comment:
This would be better put at around Line 94 as general best practice. This will ensure that no matter how the code changes below if it ever does these values will always be initialized. 

Thanks for the fix!

Reply:
You can also then remove my defaults at Lines 98 / 104 / 110 as well.

Reply:
Fixed